### ISSUE TITLE
do not allow renaming a file while upload in progress 

### ISSUE THREAD

**Issue by eyaler** [googlecolab/colabtools#1158]
renaming a file while an upload is in progress will leave you with two silently broken files. maybe also relevant for move.

**Comment by eyaler** [comment]
click on upload button. upload a large file. during upload is in progress. refresh the file view and click on the filename to rename it. wait for upload to finish. you now have two broken files.

**Comment by craigcitro** [comment]
Yeah, that's a nasty state of affairs; tracked internally as b/155431361.
